servicetest/testframework/testhelper.py

```diff
@@ -150,14 +198,48 @@ def __file_name(self):
                         metavar="path",
                         help=get_env_vars_help_message)
 
+    do_ping_help_message = \
+    """ A path to a file in which to store the result of pinging
+        from inside the container. Option --ping-host is required to
+        be used in combination to this.
+    """
+    parser.add_argument("--do-ping",
+                        nargs=1,
+                        action="store",
+                        dest=DO_PING_OPTION,
+                        default=None,
+                        metavar="path",
+                        help=do_ping_help_message)
+
+    ping_host_help_message = \
+    """ A host name to ping
+    """
+    parser.add_argument("--ping-host",
+                        nargs=1,
+                        action="store",
+                        dest=PING_HOST_OPTION,
+                        default=None,
+                        metavar="hostname",
+                        help=ping_host_help_message)
+
     args = parser.parse_args()
 
-    file_base_path_parsed_value = getattr(args, GET_ENV_VARS_OPTION)
-    if file_base_path_parsed_value is not None:
+    env_vars_file_base_path_parsed_value = getattr(args, GET_ENV_VARS_OPTION)
+    if env_vars_file_base_path_parsed_value is not None:
         # Extract the actual path string
-        file_base_path = file_base_path_parsed_value.pop()
-        h = EnvironmentHelper(file_base_path)
+        env_vars_file_base_path = env_vars_file_base_path_parsed_value.pop()
+        h = EnvironmentHelper(env_vars_file_base_path)
         # Get the environment from inside the container
         env_vars = h.get_env_vars()
         # Dump the information for the helper to read back later in the tests
         h.write_result(env_vars)
+
+    do_ping_file_base_path_parsed_value = getattr(args, DO_PING_OPTION)
+    if do_ping_file_base_path_parsed_value is not None:
```

Comment:
I agree. I think this structure will need to change anyway, when and if we can make use of this generic helper design. But point taken, I will try to find some better names or use whitespace for an easier time for the eye.